components/automate-ui/src/app/modules/infra-proxy/cookbook-details/cookbook-details.component.spec.ts

```diff
@@ -84,7 +84,7 @@ describe('CookbookDetailsComponent', () => {
       state: {
         ...defaultRouterRouterState,
         url: `infrastructure/chef-servers/${serverId}/org/${orgId}/cookbooks/${cookbook_name}`,
-        params: { id: serverId, orgid: orgId, cookbook_name: cookbook_name }
+        params: { id: serverId, 'org-id': orgId, 'cookbook-name': cookbook_name }
```

Comment:
But here you have an inconsistency, "orgId" with camelcase and "cookbook_name" with snake case. Could we settle on one of those? Or, is there a reason you cannot just use "org-id" and "cookbook-name" without remapping them?

Reply:
@msorens For now settled with snake case :)